src/routes/pat/pat.ts

```diff
@@ -0,0 +1,60 @@
+import { sendError } from '@/errors';
+import { getUser, gqlSdk } from '@/utils';
+import { RequestHandler } from 'express';
+import Joi from 'joi';
+import { v4 as uuidv4 } from 'uuid';
+
+export const createPATSchema = Joi.object({
+  expiresAt: Joi.date().required(),
+  metadata: Joi.object(),
+}).meta({ className: 'CreatePATSchema' });
+
+export const createPATHandler: RequestHandler<
+  {},
+  {},
+  { metadata: object; expiresAt: Date }
+> = async (req, res) => {
+  if (!req.auth) {
+    return sendError(res, 'unauthenticated-user');
+  }
+
+  const { userId } = req.auth as RequestAuth;
+
+  const user = await getUser({ userId });
+
+  if (!user) {
+    return sendError(res, 'invalid-refresh-token');
```

Comment:
did you mean user instead of refresh token here?

Reply:
Good catch. Will fix this.